src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -14,18 +14,22 @@ import CurrentUserRoute from "../../../relay/queries/current_user"
 import ThreewThankYou from "../containers/3w_thank_you"
 import AcbThankYou from "../containers/acb_thank_you"
 import InquiriesContainer from "../containers/inquiries"
+import RepeatVisitor from "../containers/repeat_visitor"
 
 import { markCollectorAsLoyaltyApplicant } from "./gravity"
 
 export function Home(req: Request, res: Response, next: NextFunction) {
   return res.redirect(req.baseUrl + "/inquiries")
 }
 
-export function ThankYouHtml(info: CollectorProfileResponse, userName?: string): string {
-  if (info.confirmed_buyer_at) {
-    return renderToString(<AcbThankYou />)
+export function ThankYouHtml(info: CollectorProfileResponse, userName?: string, recentApplicant?: boolean): string {
```

Comment:
I know you're exporting this function for tests but since it isn't a route handler, we should think about moving it to another file to avoid confusion